releasenotes/notes/enable-imdsv2-default-7be51aed461356e0.yaml

```diff
@@ -0,0 +1,14 @@
+---
+upgrade:
+  - |
+    The setting 'ec2_prefer_imdsv2' now defaults to `true`. The Agent will now
+    try to fetch a token from the IMDSv2 API by default to request EC2
+    metadata. When IMDSv2 is not available, the agent still automatically falls
+    back on IMDSv1.
+    Upon upgrading, this will change the hostname used by the Agent in case
+    where IMDSv2 was enforced on AWS side but `ec2_prefer_imdsv2` was left to
+    `false` (the Agent used to fail fetching the instance ID from IMDSv1 and
```

Comment:
```suggestion
    Upon upgrading, this will change the hostname used by the Agent in cases
    where IMDSv2 was enforced on the AWS side but `ec2_prefer_imdsv2` was left to
    `false` (the Agent used to fail fetching the instance ID from IMDSv1 and
```